plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/androidx/room/visitor/AbstractRoomVisitor.kt

```diff
@@ -0,0 +1,42 @@
+package io.sentry.android.gradle.instrumentation.androidx.room.visitor
+
+import io.sentry.android.gradle.instrumentation.AbstractSpanAddingMethodVisitor
+import org.objectweb.asm.Label
+import org.objectweb.asm.MethodVisitor
+
+abstract class AbstractRoomVisitor(
```

Comment:
this is only DRY right? same code as before + `className`

Reply:
yep